### ISSUE TITLE
HL Minimap Toggle - Right Click Function

### ISSUE THREAD

**Issue by dsblack115** [nmsmith22389/HotLoot#1]
![wowscrnshot_101416_203004](https://cloud.githubusercontent.com/assets/22847171/19406073/52b0c22c-924d-11e6-8978-19c498a561f8.jpg)

I have played for about an hour and a half, have not run into any issues as far as errors are concerned, but I have found a feature which does not work, and it is the right click function to display history as shown in the screenshot.  Not that important in my opinion, but out of curiosity I tried to see what it would do, or what the history would look like.  It did not do anything.  The left click, and mouse wheel functions work properly.

**Comment by nmsmith22389** [comment]
Yeah at one point there was a loot history feature that was moved to an extra package that’s now abandoned... At some point ill probably put it back in. I guess that was just left over from when it used to be in there and I forgot to take it out.

I'll remove it in the next update.

Thanks for finding it!

**Comment by dsblack115** [comment]
No problem!  Makes sense.  So far, so good.  I did forget to get the BugGrabber/BugSack addons, but I just did a brief looting run after adding them, no bugs found.  This may seem petty, but the only feature I can really think of at the moment, and this is mainly because I turned on all of the features manually, maybe a select all function in the selection menus?

My apologies as I am new to this, so I am not sure where to put feature requests.
